jeremy2918/tsseract
# Code refactor

# Code refactor prior lunch.


This project was started with no idea to ever publish it, but I will do now. So lets get it done.

## Testing Steps

- Pull and checkout to this branch.
- `npm install`
- `npm run tsc`
- `npm test` All tests should be passing successfully.
- `npm run watch`
